open-company/open-company-web
# UX desktop: drafts click

Card: https://trello.com/c/OoJc5Qtd

To test:
- [x] expand a post, did it work? Good
- click on New
- add a long body
- do NOT publish it, close Cmail
- click New again
- add a short body (1 line)
- do NOT publish it, close Cmail
- go to drafts
- click on the first draft
- [x] did it open in edit mode? Good
- click on the second
- [x] did it open in edit mode? Good
- now publish both
- go to AP
- click on both bodies
- [x] does the long one expanded? Good
- [x] do you see your view in WST? Good
